src/core_landice/Registry.xml

```diff
@@ -172,6 +180,46 @@
                         description="If true, then ice from small grounded islands is removed.  Specifically, this finds one- and two-cell masses of ice that are surrounded by open ocean and eliminates them by sending them to the calving flux.  Grounded such ice masses would be islands, floating would be icebergs.  The ice removed is added to the calvingThickness field."
 	                possible_values=".true. or .false."
 		/>
+		<nml_option name="config_finalize_damage_after_advection" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the li_finalize_damage_after_advection function."
```

Comment:
I change default value to true and add another sentence to explain why

Reply:
@tongzhangice , I don't see the changes you describe in the PR.  Did you push the commit with these changes?

Reply:
@tongzhangice -- I update this based on your comment above. Please have a look at my description in the registry now and make sure you agree with it, particularly w.r.t. what the preserve damage option implies when the 'li_finalize_damage_after_advection' subroutine is called. Your description in that subroutine is al little bit unclear. Please confirm if the only thing this option does is prevent healing to occur (it could also be inferred to prevent damage from going over a defined threshold value -- or does it do both?). We need to make it clear if the only thing this does is negate any healing that occurs over a time step or if it does other things as well.